addons/account_edi/models/account_edi_document.py

```diff
@@ -16,7 +17,7 @@ class AccountEdiDocument(models.Model):
     move_id = fields.Many2one('account.move', required=True, ondelete='cascade')
     edi_format_id = fields.Many2one('account.edi.format', required=True)
     attachment_id = fields.Many2one('ir.attachment', help='The file generated by edi_format_id when the invoice is posted (and this document is processed).')
-    state = fields.Selection([('to_send', 'To Send'), ('sent', 'Sent'), ('to_cancel', 'To Cancel'), ('cancelled', 'Cancelled')])
+    state = fields.Selection([('to_send', 'Sending'), ('sent', 'Sent'), ('to_cancel', 'Cancelling'), ('cancelled', 'Cancelled')])
```

Comment:
you can't do without adding these new words into the `.pot` for translations but for the purpose of this backport, I suggest to let that as before